apple/build-xcframework.sh

```diff
@@ -13,6 +13,7 @@ for sdk in macosx iphoneos iphonesimulator; do
     BUILD_LIBRARY_FOR_DISTRIBUTION=YES
 done
 
+rm -rf ./Connlib.xcframework
```

Comment:
Because without that, I observed that `xcodebuild -create-xcframework` says:
~~~
“connlib.framework” couldn’t be copied to “macos-arm64_x86_64” because an item with the same name already exists
~~~